pages/account/connected-accounts/index.md

```diff
@@ -1,14 +0,0 @@
----
-title: Connected Accounts
-layout: page-react.html
-entryname: connected-accounts
-vagovstaging: true
-vagovprod: true
----
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs" id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/" onClick="recordEvent({ event: 'nav-breadcrumb', 'nav-breadcrumb-section': 'home' });">Home</a></li>
-    <li><a href="/account/">Your Account Settings</a></li>
-    <li><a aria-current="page" href="/account/connected-accounts/">Connected Accounts</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-edc1946e05f624cbcdc087c39d5f66c4R7.